plugins/BUILD

```diff
@@ -1,7 +1,8 @@
 plugin_repo(
     name = "go",
     plugin = "go-rules",
-    revision = "v1.20.1",
+    revision = "02f5910e44bb8274b5bdb80ec80188a9d180f59f",
```

Comment:
Need to revert this to the official go-rules before merging.

Reply:
Yup thanks, have done - wanted to get a build on here against it to check that really did fix it.